src/main/kotlin/snyk/container/ContainerService.kt

```diff
@@ -49,16 +48,41 @@ class ContainerService(project: Project) : CliAdapter<ContainerResult>(
             "container scan: images with vulns [${tempResult.allCliIssues?.size}], issues [${tempResult.issuesCount}] "
         )
 
+        val images = imageCache?.getKubernetesWorkloadImages() ?: emptySet()
         tempResult.allCliIssues?.forEach { forImage ->
             val baseImageRemediationInfo = convertRemediation(forImage.docker.baseImageRemediation)
+            val sanitizedImageName = sanitizeImageName(forImage.imageName, imageNames)
             val enrichedContainerIssuesForImage = forImage.copy(
-                workloadImages = images.filter { it.image == forImage.imageName },
+                imageName = sanitizedImageName,
+                workloadImages = images.filter { it.image == sanitizedImageName },
                 baseImageRemediationInfo = baseImageRemediationInfo
             )
-            containerIssueImageList.add(enrichedContainerIssuesForImage)
+            patchedIssueImageList.add(enrichedContainerIssuesForImage)
         }
         LOG.debug("container service scan completed")
-        return ContainerResult(containerIssueImageList, null)
+        return ContainerResult(patchedIssueImageList, null)
+    }
+
+    /**
```

Comment:
We should ask them again - I had done so a few weeks ago, but didn't get a reply.

Reply:
it will take time on their side anyway. So we need some workaround for now...